src/components/tasks/TaskCommandList.tsx

```diff
@@ -58,16 +58,36 @@ class TaskCommandList extends React.Component<Props> {
     };
     let finished = command.durationInSeconds > 0 || isTaskCommandFinalStatus(command.status);
     let expandable = command.name === selectedCommandName || finished || !isTaskFinalStatus(this.props.task.status);
+
+    // the text at the top (name and type)
+    let topText: JSX.Element;
+
+    if (command.type === 'CLONE') {
+      topText = <Typography variant="body1">Clone</Typography>;
+    } else if ((['EXECUTE_SCRIPT', 'EXECUTE_BACKGROUND_SCRIPT'] as Array<TaskCommandType>).includes(command.type)) {
+      topText = <Typography variant="body1">Run {command.name}</Typography>;
+    } else if (command.type === 'CACHE') {
+      topText = <Typography variant="body1">Populate {command.name} cache</Typography>;
+    } else if (command.type === 'UPLOAD_CACHE') {
+      // the upload text is added on the backend
+      topText = <Typography variant="body1">{command.name}</Typography>;
+    } else if (command.type === null) {
+      // todo: remind Fedor to fix this on the backend
+      topText = <Typography variant="body1">Upload {command.name} artifacts</Typography>;
+    } else {
```

Comment:
I see the same `<Typography variant="body1">` in each case, so maybe we should take it out? And assign only different parts to `topText`. And wrap the `topText` usage into `<Typography variant="body1">`.